integration/e2e/service.go

```diff
@@ -183,13 +192,17 @@ func (s *ConcreteService) Endpoint(port int) string {
 	return fmt.Sprintf("127.0.0.1:%d", localPort)
 }
 
-func (s *ConcreteService) NetworkEndpoint(networkName string, port int) string {
-	return fmt.Sprintf("%s:%d", s.containerName(networkName), port)
+func (s *ConcreteService) NetworkEndpoint(port int) string {
+	return fmt.Sprintf("%s:%d", s.containerName(), port)
```

Comment:
I think this could be:
`return s.NetworkEndpointFor(s.usedNetworkName, port)`

Reply:
:+1: